.github/workflows/ci.yml

```diff
@@ -31,6 +31,7 @@ jobs:
       - name: gradle caching
         uses: gradle/actions/setup-gradle@v3
         with:
+          cache-encryption-key: Da25KUVSE5jbGds2zXmfXw==
```

Comment:
May have to generate a secret key for this.

https://github.com/gradle/actions/blob/main/docs/setup-gradle.md#saving-configuration-cache-data

Reply:
I prefer to not use configuration cache on CI. Our CI builds are spending way way less than 0.1% of their time on configuration, we're opening ourselves up to a bunch of pain for the chance at an 0.1% speedup (and I don't think we actually get a speedup on CI). I think it only makes sense for developer machines.

Reply:
Adding `--no-configuration-cache` all over the place is a pain, and ideally you want whatever happens on CI to match what happens on dev machines. So I'm okay with what it's doing right now

<img width="606" alt="image" src="https://github.com/diffplug/spotless/assets/2924992/5c292c4f-59b1-41c2-a75b-534826b1bdcb">

but I don't want to set a `cache-encryption-key` or manage another secret.

Reply:
Got it!